pkg/services/system_intakes_test.go

```diff
@@ -42,74 +42,41 @@ func (s *ServicesTestSuite) TestFetchSystemIntakes() {
 	}
 	fnAllFail := func(ctx context.Context) (models.SystemIntakes, error) { return nil, errors.New("forced error") }
 
-	fnByFilter := func(ctx context.Context, statuses []models.SystemIntakeStatus) (models.SystemIntakes, error) {
-		return models.SystemIntakes{
-			models.SystemIntake{},
-		}, nil
-	}
-	fnByFilterFail := func(ctx context.Context, statuses []models.SystemIntakeStatus) (models.SystemIntakes, error) {
-		return nil, errors.New("forced error")
-	}
-
 	testCases := map[string]struct {
-		ctx    context.Context
-		fn     func(context.Context, models.SystemIntakeStatusFilter) (models.SystemIntakes, error)
-		filter models.SystemIntakeStatusFilter
-		fail   bool
+		ctx  context.Context
+		fn   func(context.Context) (models.SystemIntakes, error)
+		fail bool
 	}{
 		"happy path requester": {
 			appcontext.WithPrincipal(context.Background(), requester),
-			NewFetchSystemIntakes(serviceConfig, fnByID, fnAllFail, fnByFilterFail, fnAuth),
-			models.SystemIntakeStatusFilter(""),
+			NewFetchSystemIntakes(serviceConfig, fnByID, fnAllFail, fnAuth),
 			false,
 		},
 		"happy path reviewer fetch all": {
 			appcontext.WithPrincipal(context.Background(), reviewer),
-			NewFetchSystemIntakes(serviceConfig, fnByIDFail, fnAll, fnByFilterFail, fnAuth),
-			models.SystemIntakeStatusFilter(""),
-			false,
-		},
-		"happy path reviewer filter": {
-			appcontext.WithPrincipal(context.Background(), reviewer),
-			NewFetchSystemIntakes(serviceConfig, fnByIDFail, fnAllFail, fnByFilter, fnAuth),
-			models.SystemIntakeStatusFilterOPEN,
+			NewFetchSystemIntakes(serviceConfig, fnByIDFail, fnAll, fnAuth),
 			false,
 		},
 		"fail authorization": {
 			context.Background(),
-			NewFetchSystemIntakes(serviceConfig, fnByID, fnAll, fnByFilter, fnAuth),
-			models.SystemIntakeStatusFilter(""),
+			NewFetchSystemIntakes(serviceConfig, fnByID, fnAll, fnAuth),
 			true,
 		},
 		"fail requester data access": {
 			appcontext.WithPrincipal(context.Background(), requester),
-			NewFetchSystemIntakes(serviceConfig, fnByIDFail, fnAll, fnByFilter, fnAuth),
-			models.SystemIntakeStatusFilter(""),
+			NewFetchSystemIntakes(serviceConfig, fnByIDFail, fnAll, fnAuth),
 			true,
 		},
 		"fail reviewer fetch all data access": {
 			appcontext.WithPrincipal(context.Background(), reviewer),
-			NewFetchSystemIntakes(serviceConfig, fnByID, fnAllFail, fnByFilter, fnAuth),
-			models.SystemIntakeStatusFilter(""),
-			true,
-		},
-		"fail reviewer filter data access": {
```

Comment:
double check these removed tests

Reply:
Not super worried about these tests since they're all for `services`, which are not going to be used for any intake fetching.